### DISCUSSION TITLE
How get the value of a range input and pass it to a function

### DISCUSSION THREAD

**Discussion by StefanoTesla** [alpinejs/alpine#3677] [1. Help]
I got this:
`<input @change="??" type="range" x-bind:min="swi.min" x-bind:max="swi.max" x-bind:value="swi.actualValue" class="py-2" />`

If I move the slider how can I move the new value to the @change function??

**Comment by SimoTod** [comment]
$el.value

**Reply by joshhanley** [reply]
@SimoTod speak english please 😂

<img width="369" alt="image" src="https://github.com/alpinejs/alpine/assets/882837/8478c264-3869-4b61-a3dc-abdf364894d2">
